compiler/rustc_passes/src/lang_items.rs

```diff
@@ -157,7 +158,14 @@ impl<'tcx> LanguageItemCollector<'tcx> {
             self.tcx.hir().get_by_def_id(item_def_id)
         {
             let (actual_num, generics_span) = match kind.generics() {
-                Some(generics) => (generics.params.len(), generics.span),
+                Some(generics) => (
+                    generics
+                        .params
+                        .iter()
+                        .filter(|p| !self.tcx.has_attr(p.def_id, sym::rustc_host))
```

Comment:
this can use the field on `GenericParamDefKind`

Reply:
This is `hir::Generics`, do you want me to add a field to that?

Reply:
Oh 🤦